tests/components/homewizard/test_init.py

```diff
@@ -25,13 +26,17 @@ async def test_load_unload_v1(
     await hass.config_entries.async_setup(mock_config_entry.entry_id)
     await hass.async_block_till_done()
 
+    weak_ref = weakref.ref(mock_config_entry.runtime_data)
```

Comment:
if we do want to keep this in this integration for now, should we add a comment explaining why? otherwise it would not be clear for other developers that land in this test

Reply:
epenet is going to look into add a direct test for this case in a followup PR